.github/workflows/ansible-playbooks/provision.yml

```diff
@@ -0,0 +1,178 @@
+            name: "{{ apt_deps }}"
+            state: present
+            update_cache: yes
+          when: ansible_os_family == 'Debian'
+      when:
+        - install_deps is defined
+        - install_deps | bool
+
+    - name: Install Python and pip
+      block: 
+
+        # ---------------------------------------------------------------------
+        # Ubuntu --------------------------------------------------------------
+        - name: Set up Python 3.9 on Ubuntu Jammy
+          block:
+          - name: Set up Python 3.9 repository
+            apt_repository:
+              repo: 'ppa:deadsnakes/ppa'
+
+          - name: Install Python3.9 on Ubuntu Jammy
+            ansible.builtin.package:
+              name:
+                - python3.9
+                - python3.9-distutils
+              state: present
+              update_cache: yes
+
+          - name: Change Python link Ubuntu Jammy
+            command: ln -sf /usr/bin/python3.9 /usr/bin/python3
+          when:
+            - ansible_os_family == 'Debian'
+            - ansible_distribution == "Ubuntu"
+            - ansible_distribution_release == "jammy"
+        
+        - name: Change Python link Ubuntu Xenial
+          command: ln -sf /usr/local/bin/python3.8 /usr/bin/python3
+          when:
+            - ansible_pkg_mgr == "apt"
+            - ansible_distribution == "Ubuntu"
+            - ansible_distribution_release == "xenial"
+
+        # ---------------------------------------------------------------------
+        # Pip installation ----------------------------------------------------
+
+        - stat:
+            path: /usr/bin/pip3
+          register: stat_pip3
+          when:
+            - ansible_os_family == 'Debian'
+
+        - name: Install pip Ubuntu\Debian
+          shell: curl https://bootstrap.pypa.io/get-pip.py | python3 -
+          when:
+            - ansible_os_family == 'Debian'
+            - stat_pip3.stat.exists == False
+            - ansible_distribution_release not in ['bionic']
+        
+        - name: Install pip Ubuntu Bionic/Xenial
+          ansible.builtin.package:
+            name:
+              - python3-pip
+            state: present
+            update_cache: yes
+          when:
+            - ansible_os_family == 'Debian'
+            - ansible_distribution == "Ubuntu"
+            - ansible_distribution_release in ['bionic']
+
+      when:
+        - install_python is defined
+        - install_python | bool
+
+    # No version specified in pyyaml due to Xenial error.
+    - name: Install pytest
+      command: pip3 install {{ item }}
+      with_items: "{{ pip_deps }}"
+      when:
+        - install_pip_deps is defined
+        - install_pip_deps | bool
+
+    - name: Clone installation assistant git repository
+      git:
+        repo: "{{ repository }}"
+        dest: "{{ tmp_path }}"
+        version: "{{ reference }}"
+        depth: 1
+        force: true
+
+    - name: Generate unattended
```

Comment:
Change the `unattended` references for `Installation assistant`.

Reply:
Done